packages/vx-geo/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as Albers } from './components/Albers';
+export { default as Mercator } from './components/Mercator';
+export { default as Orthographic } from './components/Orthographic';
```

Comment:
minor thing, could we rename the `./components/`dir to `./projections/`? this will help match convention with the other packages.